python/storage-initializer/scripts/initializer-entrypoint

```diff
@@ -11,4 +14,15 @@ src_uri = sys.argv[1]
 dest_path = sys.argv[2]
 
 logging.info("Initializing, args: src_uri [%s] dest_path[ [%s]" % (src_uri, dest_path))
-kfserving.Storage.download(src_uri, dest_path)
\ No newline at end of file
+
+with tempfile.TemporaryDirectory(prefix=".", dir=dest_path) as tmp_dir:
+    logging.info(f"Downloading staging directory: {tmp_dir}")
+    kfserving.Storage.download(src_uri, tmp_dir)
+    files = os.listdir(tmp_dir)
+    logging.info(f"Files downloaded: {files}")
+    logging.info(f"Copying to target location: {dest_path}")
+    for file in files:
+        os.rename(os.path.join(tmp_dir, file), os.path.join(dest_path, file))
+
+while True:
+    time.sleep(60*60)
```

Comment:
why do we need to sleep? Couldn't the storage sidecar return 0 once finishes?

Reply:
This would make it uselessly run over and over again.
In fact we might want to make sure that this has no bad consequences.

Reply:
@Gilbert88, we can not `exit` here because as storage initializer is a sidecar now, it will be restarted disregarding the exit code.

@porridge, I agree `sleep` doesn't look great however it seems to be the less intrusive, not CPU-intensive, and not over-engineered solution. E.g. IMO `sleep` is better than e.g. Python multithreading.